app/routes/_layout+/settings.account.tsx

```diff
@@ -48,22 +49,33 @@ export async function action({ context, request }: ActionFunctionArgs) {
   if (formData.get("intent") === "resetPassword") {
     const email = formData.get("email") as string;
 
-    const { error } = await sendResetPasswordLink(email);
+    const data = await db.session.findMany({
+      where: { userId: authSession.userId },
+    });
 
-    if (error) {
-      return json(
-        {
-          message: "Unable to send password reset link",
-          email: null,
-        },
-        { status: 500 }
-      );
-    }
+    const updatedData = await db.session.updateMany({
```

Comment:
shouldn't we delete all existing sessions for this userId and then create a new one + save it in the context before redirect?